testnets/_IBC/stride-gaia.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain-1": {
+    "chain-name": "stride",
```

Comment:
chain-1 and chain-2 should always be alphabetical. So gaia would be chain-1 and stride should be chain-2.